select2-parent/select2/src/main/java/org/wicketstuff/select2/Settings.java

```diff
@@ -58,7 +57,7 @@
 
 	private AjaxSettings ajax;
 	private String data;
-	private String tags;
+	private Boolean tags;
```

Comment:
I'll try to check documentation for default values and fix this (no estimates unfortunately :(( )